miio/airfresh.py

```diff
@@ -206,28 +301,7 @@ class AirFresh(Device):
     def status(self) -> AirFreshStatus:
         """Retrieve properties."""
 
-        properties = [
-            "power",
-            "temp_dec",
-            "aqi",
-            "average_aqi",
-            "co2",
-            "buzzer",
-            "child_lock",
-            "humidity",
-            "led_level",
-            "mode",
-            "motor1_speed",
-            "use_time",
-            "ntcT",
-            "app_extra",
-            "f1_hour_used",
-            "filter_life",
-            "f_hour",
-            "favorite_level",
-            "led",
-        ]
-
+        properties = AVAILABLE_PROPERTIES[self.model]
         values = self.get_properties(properties, max_properties=15)
```

Comment:
The max properties was limited in va4, was it?

Reply:
I assume it applies for both (va2 and va4).